deploy/backplane/srep/dynatrace/config.yaml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+  - key: ext-hypershift.openshift.io/cluster-type
+    operator: In
+    values: ["management-cluster","service-cluster"]
```

Comment:
There isn't some dynatrace specific label we can use?  This will work but will require changes when this stack is used beyond HCP ROSA.

Reply:
There's not, no - this is the only way we have at the moment of targeting where Dynatrace is landing, and is consistent with the same targeting we use in the SelectorSyncSets for the DT Operator and configs.

Granted, we do know that this will also ideally need to land on some AppSRE clusters too, but we do presently lack any clusterdeployment labelling that helps us better target everything.